crates/common_enums/src/enums.rs

```diff
@@ -2796,9 +2796,16 @@ pub enum PermissionGroup {
     MerchantDetailsManage,
     // TODO: To be deprecated, make sure DB is migrated before removing
     OrganizationManage,
-    ReconOps,
     AccountView,
     AccountManage,
+    ReconTokenView,
```

Comment:
Let's not create a new group for this. The `ReconToken` resource should be part of the other 2 groups.

Reply:
We will not be able to specify multiple permissions to HyperSwitch API resources, so I've added this extra permission.